hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java

```diff
@@ -47,68 +41,66 @@
 public final class ContainerBalancerConfiguration {
   private static final Logger LOG =
       LoggerFactory.getLogger(ContainerBalancerConfiguration.class);
-  private OzoneConfiguration ozoneConfiguration;
 
   @Config(key = "utilization.threshold", type = ConfigType.AUTO, defaultValue =
-      "0.1", tags = {ConfigTag.BALANCER},
-      description = "Threshold is a fraction in the range of 0 to 1. A " +
+      "10", tags = {ConfigTag.BALANCER},
+      description = "Threshold is a percentage in the range of 0 to 100. A " +
           "cluster is considered balanced if for each datanode, the " +
           "utilization of the datanode (used space to capacity ratio) differs" +
           " from the utilization of the cluster (used space to capacity ratio" +
-          " of the entire cluster) no more than the threshold value.")
-  private String threshold = "0.1";
+          " of the entire cluster) no more than the threshold.")
+  private String threshold = "10";
```

Comment:
can we also make threshold int not String?

Reply:
Yes that's what I wanted to do. But we'll need to deprecate threshold in the proto message and add a new field with the int32 type. Should I go ahead with that?